saleor/plugins/webhook/plugin.py

```diff
@@ -1099,7 +1100,10 @@ def __run_payment_webhook(
             )
 
         webhook_payload = generate_payment_payload(payment_information)
-        response_data = trigger_webhook_sync(event_type, webhook_payload, app)
+        payment = Payment.objects.get(id=payment_information.payment_id)
```

Comment:
It is a detail but wouldn't be better to use `filter().first()` here and confirm that `payment` != None? I know that this is almost not possible but  this would safe us from case when `.get` raises an exception

Reply:
good point, fixed